src/pages/landing/deriv-go/_other-apps.js

```diff
@@ -60,7 +61,7 @@ const Card = styled(Flex)`
     padding: 80px 24px;
     border-radius: 12px;
     box-shadow: 0 16px 40px rgba(198, 198, 198, 0.3);
-    margin-right: 2.4rem;
+    margin-right: 2.5rem;
     margin-bottom: 2rem;
```

Comment:
Any reason you're using `px` above but `rem` here?

Reply:
no particular reason